simulation/sub8_gazebo/launch/duck.launch

```diff
@@ -2,7 +2,7 @@
   <param name="simulate" value="true" />
```

Comment:
Hmm I was having issues with TF without it that were solved by adding the line.
Empty_world.launch would set it but I removed the call to that file. I think we need both.